src/mercury.c

```diff
@@ -22,6 +22,10 @@
 #include <string.h>
 #include <assert.h>
 
+/* PVAR profiling support */
+#include "mercury_prof_pvar_impl.h"
+HG_PROF_PVAR_UINT_COUNTER_DECL_EXTERN(hg_pvar_hg_forward_count);
+
```

Comment:
As a general comment I think it would be good to see if/how we can avoid declaring global variables.